cmd/snap/cmd_model.go

```diff
@@ -0,0 +1,229 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/client"
+	"github.com/snapcore/snapd/i18n"
+)
+
+var (
+	shortModelHelp = i18n.G("Get the active model for this device")
+	longModelHelp  = i18n.G(`
+The model command returns only the active model assertion information for this 
+device.	
+
+The --serial option allows printing the serial assertion instead of the model
```

Comment:
If the text is mostly repeating the same information, we don't normally describe additional arguments outside of the command options section. Maybe change this to a more descriptive paragraph, even listing the headers that are included by default and where any additional headers come from, along the lines of:

By default, only a minimal set of headers are included in the output, but this can be expanded to include all of an assertion's headers. Similarly, the default model assertion can alternatively be replaced with the serial assertion.

Reply:
Okay, I took what you provided and slightly reworded it since we aren't printing all assertion headers with --verbose but rather all of the non-meta assertion headers (i.e. no headers that are about the assertion itself like the body length, etc.)